doc/source/configure.rst

```diff
@@ -162,6 +163,19 @@ when was the last data sent, and other internal information.
 The **Log** section contains a list of log messages
 that the agent sent to QAN API.
 
+Configuring Metrics Monitor
+===========================
+
+There are standard Grafana settings
+that you can access using the **Manage Dashboards** gear icon
+in the header toolbar.
+
+Prometheus web interface can be accessed by adding ``/prometheus/``
+to the PMM server address.
+
+Consul web interface can be accessed by adding ``/consul/``
```

Comment:
I would add a warning here that it is not supposed to manage PMM from Consul web interface, for example, an user can delete nodes from there and it will stop the monitoring. Users have to use `pmm-admin` and Consul UI is only for visibility.

Unlike Prometheus UI, an user can stop monitoring from Consul UI, a friendly warning :D